gc/base/GCExtensionsBase.hpp

```diff
@@ -257,6 +257,7 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 	bool largePageWarnOnError;
 	bool largePageFailOnError;
 	bool largePageFailedToSatisfy;
+	bool isDoubleMapRequested; /** Value of -XXgc:enableDoubleMapping <true> or -XXgc:disableDoubleMapping <false> specified by the user */
```

Comment:
should not really refer to option name directly (they are specific for OpenJ9). let alone (in theory) they can be even set by some other means like shell environmental variables, property files etc.